pkg/crc/machine/stop.go

```diff
@@ -52,10 +52,9 @@ func removeMCOPods(host *host.Host, client *client) error {
 	}
 	defer sshRunner.Close()
 
-	ocConfig := oc.UseOCWithSSH(sshRunner)
-	_, stderr, err := ocConfig.RunOcCommand("delete", "pods", "--all", "-n openshift-machine-config-operator", "--grace-period=0")
-	if err != nil {
-		logging.Debugf("Error deleting the pods from openshift-machine-config-operator namespace:%v \n StdErr: %s", err, stderr)
+	if err := systemd.NewInstanceSystemdCommander(sshRunner).Stop("kubelet"); err != nil {
+		return err
 	}
-	return nil
+	_, _, err = sshRunner.RunPrivileged("", `-- sh -c 'crictl stop $(crictl ps -q)'`)
```

Comment:
@guillaumerose I would like to capture the `stderr` here and return it along with `err` because sometime error message is very vague and doesn't give any context so better to add stderr also.